shopping-list-web/src/common/autocompleteDropdown.tsx

```diff
@@ -0,0 +1,37 @@
+import "./autocompleteDropdown.css";
+
+const AutocompleteDropdown = (props: {
```

Comment:
think you can do `export default function AutocompleteDropdown(` and then you don't need to export it at the bottom